changelog/README.rst

```diff
@@ -0,0 +1,35 @@
+:orphan:
+
+Changelog
+=========
+
+This directory contains "news fragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next what's new page.
+
+Make sure to use full sentences with correct case and punctuation, and please
+try to use Sphinx intersphinx using backticks.
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
+
+* ``api_chage``: A change which requires users to change code and is not
+  backwards compatible. (Not to be used for removal of deprecated features.)
+* ``feature``: New user facing features and any new behavior.
```

Comment:
I like the `attrs` style that calls these `breaking` and `change`. I think the `api_change` and `feature` distinction is roughly the same, but I think people will not be super clear on what to do about bug fixes.

Reply:
Thanks for the feedback! I've changed the categories a bit along these lines.